### ISSUE TITLE
ModifiedAt changes with the same attribute value

### ISSUE THREAD

**Issue by sgonzalez-hiades** [Fiware/context.Orion-LD#1481]
Performing tests we have detected the following: we are using the endpoint `/ngsi-ld/v1/entityOperations/upsert?options=update` and in the body of the request there is a attribute that has the same value that it had previously, this causes the value of `modifiedAt` of the attribute to be updated. We have realized that the same doesn't happen with the `location` property, since `modifiedAt` is not updated with the same value.

This is a problem when generating notifications because the attributes values haven't really changed but the notification will be trigger because the `modifiedAt` metadata was modified.

**Comment by kzangeli** [comment]
Yeah well, every time an update it sent to the broker, the modifiedAt must be changed.
It's a new "time of sample", so that's the way it must work.
If that is not done with the "location" GeoProperty ... that's a BUG and I'll look into that.

Now, about not wanting to receive notifications for updates that didn't ACTUALLY change the value of an attribute is not new and there's something implemented in Orion-LD for that:
Start the broker with the CLI `-noNotifyFalseUpdate`  and you should no longer receive those notifications.
Env var: `export ORIONLD_NO_NOTIFY_FALSE_UPDATE=TRUE`

**Comment by kzangeli** [comment]
Note to self:  Need to look at `modifiedAt` for "location"
